apps/o2-blocks/src/github-issue-template/editor.js

```diff
@@ -84,15 +79,14 @@ const Invalid = () => (
 		className="is-warning"
 		title={ __( 'Please fill in the required fields' ) }
 		subTitle={ __(
-			'Title, Org, and Repository are required. Select this block to open the form and fill them.'
+			'Org and Repository are required. Select this block to open the form and fill them.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.